README.md

```diff
@@ -1 +1,170 @@
-# wdltool
\ No newline at end of file
+Command line utilities for interacting with WDL
+
+<!---toc start-->
+
+* [Requirements](#requirements)
+* [Building](#building)
+* [Command Line Usage](#command-line-usage)
+  * [validate](#validate)
+  * [inputs](#inputs)
+  * [parse](#parse)
+  * [highlight](#highlight)
+* [Getting Started with WDL](#getting-started-with-wdl)
+
+<!---toc end-->
+
+# Requirements
+
+The following is the toolchain used for development of Cromwell.  Other versions may work, but these are recommended.
+
+* [Scala 2.11.7](http://www.scala-lang.org/news/2.11.7)
+* [SBT 0.13.8](https://github.com/sbt/sbt/releases/tag/v0.13.8)
+* [Java 8](http://www.oracle.com/technetwork/java/javase/overview/java8-2100321.html)
+
+# Building
+
+`sbt assembly` will build a runnable JAR in `target/scala-2.11/`
+
+Tests are run via `sbt test`.  Note that the tests do require Docker to be running.  To test this out while downloading the Ubuntu image that is required for tests, run `docker pull ubuntu:latest` prior to running `sbt test`
+
+# Command Line Usage
+
+Run the JAR file with no arguments to get the usage message:
+
+```
+$ java -jar cromwell.jar
+java -jar cromwell.jar <action> <parameters>
+
+Actions:
+parse <WDL file>
+
+  Compares a WDL file against the grammar and prints out an
+  abstract syntax tree if it is valid, and a syntax error
+  otherwise.  Note that higher-level AST checks are not done
+  via this sub-command and the 'validate' subcommand should
+  be used for full validation
+
+validate <WDL file>
+
+  Performs full validation of the WDL file including syntax
+  and semantic checking
```

Comment:
If it is removed maybe it could find a new home in verbose output of the `validate` command